### PR TITLE
Fixes for aws blocks

### PR DESCRIPTION
**Pull request by SeanTConrad**
AWS blocks wasn't working. Testing with TL account. Made some fixes.

### CODE REVIEW

File: src/verinfast/cloud/aws/blocks.py [hunk 1]
```diff
@@ -31,27 +31,35 @@ def getBlocks(sub_id: str, path_to_output: str = "./"):
...
+
+            if "Policy" in policy_status_resp:
+                p_string = policy_status_resp["Policy"]
+                p_dict = json.loads(p_string)
+                statements = p_dict["Statement"]
+                # print(statements)
+                permissions = [json.dumps(s) for s in statements]
+                # print(s2)
+        except s3.exceptions.from_code('NoSuchBucketPolicy'):
+            print("No Bucket Policy for bucket " + bucket_name)
```
Comment (aylusltd) [inline src/verinfast/cloud/aws/blocks.py:52]: Do we need to print? I don't think user cares.

**Review by aylusltd — APPROVED** [review]
Looks good to me.

**Comment by SeanTConrad** [comment]
Made a few more changes to this @aylusltd

File: .flake8 [hunk 2]
```diff
@@ -2,6 +2,7 @@
 per-file-ignores =
     # line too long
     src/verinfast/agent.py: E501, E722, W605
+    src/verinfast/cloud/aws/blocks.py: E501
```
Comment (aylusltd) [inline .flake8:5]: Prefer where possible to break up the long lines, but will merge this one.

**Pull request merged**